taskcluster/ci/config.yml

```diff
@@ -35,24 +35,22 @@ workers:
             implementation: docker-worker
             os: linux
             worker-type: 'b-linux'
-        images:
-            provisioner: 'mobile-{level}'
-            implementation: docker-worker
-            os: linux
-            worker-type: 'images'
         dep-signing:
             provisioner: scriptworker-k8s
             implementation: scriptworker-signing
             os: scriptworker
             worker-type: mobile-t-signing
-        signing:
-            provisioner: scriptworker-k8s
-            implementation: scriptworker-signing
-            os: scriptworker
-            worker-type:
-                by-level:
-                    "3": mobile-3-signing
-                    default: mobile-t-signing
+        images:
+            provisioner: 'mobile-{level}'
+            implementation: docker-worker
+            os: linux
+            worker-type: 'images'
+        # misc is used by taskgraph to generate tasks with more than 10 routes
+        misc:
```

Comment:
This change and the taskgraph bump are the only differences with the original patch https://github.com/mozilla-mobile/fenix/pull/10092